kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/ProxyChannelStateMachine.java

```diff
@@ -105,17 +110,35 @@
 public class ProxyChannelStateMachine {
     private static final String DUPLICATE_INITIATE_CONNECT_ERROR = "NetFilter called NetFilterContext.initiateConnect() more than once";
     private static final Logger LOGGER = getLogger(ProxyChannelStateMachine.class);
+    @Deprecated(since = "0.13.0", forRemoval = true)
     private final Counter downstreamConnectionsCounter;
+    @Deprecated(since = "0.13.0", forRemoval = true)
     private final Counter upstreamConnectionsCounter;
+    @Deprecated(since = "0.13.0", forRemoval = true)
     private final Counter downstreamErrorCounter;
+    @Deprecated(since = "0.13.0", forRemoval = true)
     private final Counter upstreamErrorCounter;
+    @Deprecated(since = "0.13.0", forRemoval = true)
     private final Counter connectionAttemptsCounter;
+    @Deprecated(since = "0.13.0", forRemoval = true)
     private final Counter upstreamConnectionFailureCounter;
+    private final Counter clientToProxyErrorCounter;
+    private final Counter clientToProxyConnectionCounter;
+    private final Counter proxyToServerConnectionCounter;
+    private final Counter proxyToServerErrorCounter;
 
-    public ProxyChannelStateMachine(String clusterName) {
-        List<Tag> tags = Metrics.tags(Metrics.VIRTUAL_CLUSTER_TAG, clusterName);
+    public ProxyChannelStateMachine(String clusterName, @Nullable Integer nodeId) {
+        List<Tag> tags = Metrics.tags(Metrics.VIRTUAL_CLUSTER_LABEL, clusterName);
```

Comment:
Only the new metrics should us the `virtual_cluster` label.  The deprecated metrics should keep using `virtualCluster`.